grow/pods/document_fields.py

```diff
@@ -0,0 +1,80 @@
+"""
+Document fields for accessing the meta fields parsed from the document.
+"""
+
+from boltons import iterutils
+import re
+
+
+LOCALIZED_KEY_REGEX = re.compile('(.*)@([^@]+)$')
+
+
+class DocumentFields(object):
```

Comment:
One more thing – collection `_blueprint.yaml` files can technically have fields as well (e.g. titles, various metadata, collection-level content). So, just something to think about there.